test/distributed/pipelining/test_backward.py

```diff
@@ -49,11 +49,15 @@ def test_stage_backward(self, device):
 
         torch.testing.assert_close(grad_inputs[0], ref_x.grad)
 
+        rtol, atol = None, None
```

Comment:
Could you submit a new PR for test_backward.py and test_microbatch.py?

Reply:
I agree.